image_view/src/nodes/video_recorder.cpp

```diff
@@ -107,9 +105,7 @@ int main(int argc, char** argv)
 
     image_transport::ImageTransport it(nh);
     std::string topic = nh.resolveName("image");
-    image_transport::CameraSubscriber sub_camera = it.subscribeCamera(topic, 1, &callback);
-    image_transport::Subscriber sub_image = it.subscribe(topic, 1,
-            boost::bind(callback, _1, sensor_msgs::CameraInfoConstPtr()));
+    ros::Subscriber sub_image = nh.subscribe(topic, 1, &callback);
```

Comment:
you still want to use an image_transport::Subscriber so that you can view compressed images. Agree on removing CameraInfo though.